tempest/api/workloadmgr/workload/test_workload.py

```diff
@@ -652,12 +669,24 @@ def test_12_create_scheduled_workload(self):
             self.schedule = self.getSchedulerStatus(self.wid)
             LOG.debug("Workload schedule: " + str(self.schedule))
             if(self.schedule):
-                reporting.add_test_step("Verification", tvaultconf.PASS)
+                reporting.add_test_step("Verification for workload schedule", tvaultconf.PASS)
                 LOG.debug("Workload schedule enabled")
             else:
-                reporting.add_test_step("Verification", tvaultconf.FAIL)
+                reporting.add_test_step("Verification for workload schedule", tvaultconf.FAIL)
                 LOG.error("Workload schedule not enabled")
+        
+            # Delete workload
+            self.workload_delete(self.wid)
```

Comment:
When you are explicitly deleting the workload, you need to pass workload_cleanup=False in workload_create() call.

Reply:
Added change